Smooch.postman_collection.json

```diff
@@ -8792,7 +8867,7 @@
 							},
 							"response": [
 								{
-									"name": "Action: quick reply Example",
+									"name": "Action:  share Example",
```

Comment:
How come this is still in here? Wasn't this a Facebook-only feature, that Facebook has since deprecated?

Reply:
It's not actually.. that's Hyjaz's commit. In his PR, the requests got shuffled around and the diff is all kinda weird unfortunately.

Reply:
it's NOT in there? it's added on line 8870...
Could you delete it completely when you merge in Hyjaz' changes?